test/unit/models/members.test.js

```diff
@@ -0,0 +1,119 @@
+const chai = require("chai");
+const { expect } = chai;
+
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+const members = require("../../../models/members");
+const { ROLES } = require("../../../constants/users");
+const userData = require("../../fixtures/user/user")();
+
+describe("members", function () {
+  let user;
+  beforeEach(async function () {
+    user = await addUser();
+  });
+  after(async function () {
+    await cleanDb();
+  });
+
+  describe("fetchUsers", function () {
+    it("should return all the users", async function () {
+      const result = await members.fetchUsers();
+      const user = result[0];
+
+      expect(result).to.be.a("array");
+      expect(user.first_name).to.be.a("string");
```

Comment:
Looks good but as we are comparing fields directly, so we don't need to compare the field type anymore.

Also, the result looks something like this: 
![image](https://user-images.githubusercontent.com/63191499/157186806-fcecc5cc-3dcf-446c-b7fc-07bd66201ecb.png)

We can use `Object.key` with `map` function to compare all fields easily as done in `models/users.js`